### PR TITLE
Add exception handling feature

### PR DESCRIPTION
**Pull request by aheejin**
This only adds the feature and its flag and not the instructions yet.

### CODE REVIEW

File: src/wasm-features.h [hunk 1]
```diff
@@ -102,6 +108,9 @@ struct FeatureSet {
     if (hasSIMD()) {
       f(SIMD);
     }
+    if (hasExceptionHandling()) {
```
Comment (tlively) [inline src/wasm-features.h]: It's hard to tell (and is not commented!), but the features are iterated over in alphabetical order according to their LLVM name, so ExceptionHandling should be up after BulkMemory. This keeps the target features section in alphabetical order, which is nice.

File: src/wasm-binary.h [hunk 2]
```diff
@@ -846,6 +847,10 @@ enum BulkMemoryOpcodes {
   MemoryFill = 0x0b
 };
 
+enum ExceptionHandlingOpcodes {
```
Comment (tlively) [inline src/wasm-binary.h]: Should this be part of a different PR? It seems unrelated to the feature.

File: test/unit/test_features.py [hunk 3]
```diff
@@ -24,6 +24,9 @@ def check_sign_ext(self, module, error):
   def check_bulk_mem(self, module, error):
     self.check_feature(module, error, '--enable-bulk-memory')
 
+  def check_exception_handling(self, module, error):
+    self.check_feature(module, error, '--enable-exception-handling')
```
Comment (tlively) [inline test/unit/test_features.py]: Is this meant to be in preparation for adding a test in the future? I suppose adding a test would be blocked on being able to parse/print a wasm using exception handling.

Comment (aheejin) [reply test/unit/test_features.py]: Yes, I added it because now we would be able to at least pass `--enable/disable-exception-handling`, but to use it I need instructions, so it's no much use. Do you want me to remove it here?

File: src/wasm-binary.h (hunk 2 above)
Comment (aheejin) [reply src/wasm-binary.h]: Done

File: src/wasm-features.h (hunk 1 above)
Comment (aheejin) [reply src/wasm-features.h]: Done

File: test/unit/test_features.py (hunk 3 above)
Comment (tlively) [reply test/unit/test_features.py]: Seems fine to keep it in.
